components/dashboard/src/hooks/use-analytics-tracking.ts

```diff
@@ -7,9 +7,17 @@
 import { useEffect } from "react";
 import { useHistory } from "react-router";
 import { trackButtonOrAnchor, trackPathChange } from "../Analytics";
+import { useUserLoader } from "./use-user-loader";
+import { useTheme } from "../theme-context";
+import { useOrbital } from "./use-orbital";
 
 export const useAnalyticsTracking = () => {
     const history = useHistory();
+    const { user, loading } = useUserLoader();
+    const { isDark } = useTheme();
+
+    // Todo(ft): only enable on gitpod.io
+    const { orbital, isLoaded: isOrbitalLoaded, discoveryIds } = useOrbital("4aErj3uvRbye");
```

Comment:
Maybe there's a better place for configuration settings. I am just not sure what it is

Reply:
The closest I'm aware of is our server function `getStripePublishableKey()` [code](https://github.com/gitpod-io/gitpod/blob/088a77f7f1575911f6cc2970c06e168c19c0f1be/components/server/src/workspace/gitpod-server-impl.ts#L3888) that we call from [the client](https://github.com/gitpod-io/gitpod/blob/088a77f7f1575911f6cc2970c06e168c19c0f1be/components/dashboard/src/data/billing/stripe-key-query.ts#L10). This allows us to have different keys for different environments. I'm not sure if that' overkill for this, but it seems like a decent pattern. Would we ever use more than 1 space id in the dashboard for orbital?

Reply:
We would only ever have one space id for everything.

Reply:
I think for now we can leave it as-is. Thanks for sharing the code